### ISSUE TITLE
Replace goal with ComparatorStrategy

### ISSUE THREAD

**Issue by grosscol** [Display-Lab/slowmo#14]
Add `WithComparatorStrategy` predicate.  Add `ComparatorStrategy` class and defined subclasses.

Define a series of comparator strategy types:
- static value
- static percent
- static percentile
- peer mean
- achievable benchmark

Expected use:
```json
"slowmo:IsAboutMeasure":[
  { "@type": "slowmo:Measure",
    "dc:title": "",
    "identifier": "BP01",
    "slowmo:WithComparatorStrategy": { 
      "@type": "StaticPercent", 
      "@value": 95
    }
}]
```

The unspecified strategy is a placeholder for all others.  Since the values will be available from the spek, the unspecified serves as a generic

**Comment by grosscol** [comment]
Can be just `WithComparator` as psdo defines a comparator information content entity (ICE): http://purl.obolibrary.org/obo/psdo_0000093 and subclasses.

Additionally, define `ComparisonValue` as an object property of a Comparator, because the `@value` key should be used when there are other properties.
```
"slowmo:WithComparator": { ....., "slowmo:ComparisonValue": {"@value": 10} }
```

**Comment by grosscol** [comment]
Closed by 46a41a85f069b7dbcc616f1d78ea95ecb6718c71
